media/css/firefox/whatsnew/whatsnew-57.scss

```diff
@@ -0,0 +1,236 @@
+        margin: 0 auto 20px;
+        width: 220px;
+    }
+
+    h2 {
+        @include font-size-level3;
+    }
+
+    .tagline {
+        margin: 20px 0 40px;
+    }
+}
+
+@media #{$mq-phone-wide} {
+    .intro {
+        padding-bottom: 200px;
+
+        &:after {
+            @include background-size(400px 273px);
+            height: 273px;
+            margin-right: -200px;
+            width: 400px;
+            bottom: -60px;
+        }
+    }
+}
+
+@media #{$mq-tablet} {
+    .intro {
+        padding: 100px 0 60px;
+        min-height: 300px;
+
+        &:after {
+            @include background-size(600px 432px);
+            bottom: auto;
+            height: 432px;
+            margin-right: -660px;
+            top: 100px;
+            width: 600px;
+        }
+    }
+
+    .intro-content {
+        text-align: left;
+        width: 50%;
+
+        h1 {
+            @include background-size(330px 69px);
+            height: 69px;
+            margin: 0 0 40px -5px;
+            width: 330px;
+        }
+    }
+}
+
+@media #{$mq-desktop} {
+    .intro {
+        padding: 120px 0 60px;
+        min-height: 450px;
+
+        &:after {
+            @include background-size(937px 640px);
+            height: 640px;
+            margin-right: -990px;
+            width: 937px;
+        }
+    }
+
+    .intro-content {
+        width: 40%;
+        padding-left: 8%;
+    }
+}
+
+@media #{$mq-desktop-wide} {
+    .intro-content h1 {
+        @include background-size(440px 92px);
+        width: 440px;
+        height: 92px;
+    }
+}
+
+
+[dir="rtl"] {
+    .intro:after {
+        left: 50%;
+        margin-left: -140px;
+        margin-right: 0;
+        right: auto;
+    }
+
+    @media #{$mq-phone-wide} {
+        .intro:after {
+            margin-left: -200px;
+        }
+    }
+
+    @media #{$mq-tablet} {
+        .intro:after {
+            margin-left: -660px;
+        }
+
+        .intro-content {
+            text-align: right;
+
+            h1 {
+                margin: 0 -5px 40px 0;
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .intro:after {
+            margin-left: -990px;
+        }
+
+        .intro-content {
+            padding-left: 0;
+            padding-right: 8%;
+        }
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// CEO letter
+
+.quantum-letter {
+    .page-head {
+        background: #c7216a;
+        background: url('/media/img/firefox/whatsnew_57/mozillians.jpg') center bottom no-repeat,
+                    linear-gradient(148deg, #c7216a 7%, #fda41e 100%);
+        @include background-size(cover, cover);
+        min-height: 20vh;
+
+        .firefox-logo {
+            margin: 20px 0;
+        }
+
+        @media #{$mq-tall} {
+            min-height: 40vh;
+
+            .firefox-logo {
+                margin-top: 10vh;
+            }
+        }
+
+        @media (min-height: 800px) {
+            min-height: 60vh;
```

Comment:
On my laptop this value feels kinda big, with most of the letter off below screen. `40vh` feels about right though. Could this do with an additional check for a really large width?